src/odemis/dataio/hdf5.py

```diff
@@ -335,8 +335,12 @@ def _add_image_info(group, dataset, image):
             _h5svi_set_state(group["DimensionScaleY"], ST_REPORTED)
 
             # Attach the scales to each dimensions (referenced by their label)
-            dataset.dims.create_scale(group["DimensionScaleX"], "X")
-            dataset.dims.create_scale(group["DimensionScaleY"], "Y")
+            if hasattr(dataset, "make_scale"):
```

Comment:
Could you add at least at one place (ex, this line), when can we remove the `else` block? Indicate the first version of h5py which support this new function.